addons/point_of_sale/models/pos_config.py

```diff
@@ -69,7 +69,8 @@ def _get_default_tip_product(self):
         return tip_product_id
 
     name = fields.Char(string='Point of Sale', required=True, help="An internal identification of the point of sale.")
-    printer_ids = fields.Many2many('pos.printer', 'pos_config_printer_rel', 'config_id', 'printer_id', string='Order Printers')
+    printer_ids = fields.Many2many('pos.printer', string="Receipt Printers", readonly=False, store=True)
```

Comment:
`readonly` default is already False
`store` default is alreadyTrue
https://www.odoo.com/documentation/master/developer/reference/backend/orm.html?highlight=fields%20boolean#odoo.fields.Field